test/test_rosbag/bag_migration_tests/README.md

```diff
@@ -0,0 +1,349 @@
+# Bag migration tests scripts
+
+In order to generate the data and rules for bag migration tests, two scripts are provided:
+
+ * **generate_data** : Generates *.bag* files with messages (data) to run the bag migration tests. 
+ * **generate_rules** : Help in the process of creating rules between two message generations.
+
+The next sections explain how to use these scripts in detail. The rest are auxiliary scripts that should NOT be used directly.
+
+Before running any of these scripts you need to build the **test_rosbag** package at least once in your workspace and `source` the environment workspace setup file. This can be done running the following from the root of the workspace:
+
+``` bash
+$ catkin_make
+$ source devel/setup.bash
```

Comment:
That's why I started with `catkin build`, but I see your point. :smiley: 

I'd need some time to re-adapt it to `cakin_make_isolated`, but hopefully it's a single line change.